show/vlan.py

```diff
@@ -89,6 +89,18 @@ def get_proxy_arp(ctx, vlan):
 
     return proxy_arp
 
+def get_static_anycast_gateway(ctx, vlan):
+    cfg, _ = ctx
+    _, vlan_ip_data, _ = cfg
+
+    for key in vlan_ip_data:
```

Comment:
why do we need a for loop here? how about this:
```python
if vlan in vlan_ip_data:
```

Reply:
This is removed in the updated PR https://github.com/sonic-net/sonic-utilities/pull/2881